### PR TITLE
Remove unused components.

### PR DESCRIPTION
**Pull request by sri2k1us**
First round of cleanup that removes some unused components.

### CODE REVIEW

File: src/components/highlighter/Highlighter.js [hunk 1]
```diff
@@ -1,31 +0,0 @@
-import React from "react";
-import Highlight from "react-highlighter";
-import PropTypes from "prop-types";
-import { withStyles } from "@material-ui/core/styles/index";
-
-/**
- * @author aramsey
- * A function which will take a search term and some text,
- * and highlight the search term within the text
- */
```
Comment (psarando) [inline src/components/highlighter/Highlighter.js]: I thought we intended to keep this component for search results?
It's also marked as `Keep` on our clean-up spreadsheet.

**Review by ash3rz — APPROVED** [review]
Other than `Highlighter` being removed which Paul pointed out, LGTM :shipit:

File: src/components/highlighter/Highlighter.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/highlighter/Highlighter.js]: Oops! Yes. My Bad. Will restore them.

**Review by psarando — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**